src/components/app/details/appDetails/appDetails.scss

```diff
@@ -1794,11 +1794,9 @@ table.resource-tree {
 
 .source-info-container,
 .app-details-info-card {
-    .dc__app-summary__icon {
```

Comment:
Is removal of this intentional? if yes, Please verify the impact

Reply:
reverted the deletion, and handle the fix accordingly